### DISCUSSION TITLE
How to replace background color in PDF slides?

### DISCUSSION THREAD

**Discussion by zeebluejay** [ImageMagick/ImageMagick#4511] [Help]
I have slides in PDF format. They have dark blue / teal background and white text. Good for reading but not so good for printing. I'd like to convert the colors of the slides so that the text becomes black (or a dark color I choose) and the background becomes white. 

I've tried to research this and came across various posts in the older forum of Imagemagick. For example [this](https://legacy.imagemagick.org/discourse-server/viewtopic.php?t=32192). But none of the workarounds worked. For example: `convert -density 300 method.pdf -alpha copy -background '#1cb7fb' -flatten result.pdf`

I'm running macOS Big Sur on MacBok Air M1 and have installed imagemagick via home-brew. Its running v7.1.0-16.

Can I use Imagemagick to replace colors in the PDF? If yes, can you pls share the exact commands I need to run?

Thanks in advance!
ZeeKay

**Comment by fmw42** [comment]
ImageMagick is a raster processing and not good at vector formats such as PDF. It can read the PDF, but then will rasterize it and when saved as PDF, it puts the raster image into a vector shell. This is not likely what you want.
